Juice/Classes/GeneralPreferencesViewController.swift

```diff
@@ -74,4 +75,9 @@ class GeneralPreferencesViewController: NSViewController {
     @IBAction func triggerRescan(_ sender: Any) {
         preferences.scanApplicationSupportForFiles()
     }
+    
+    @IBAction func editCurrentScale(_ sender: Any) {
+        let currentScale = preferences.scales.value[statusBarStylePopUp.indexOfSelectedItem]
+        NSWorkspace.shared().open((FileManager.default.applicationSupportDirectory?.appendingPathComponent(currentScale.fileName))!)
```

Comment:
I'd like to avoid force unwrapping anything in this project, could you restructure this code with a `guard` to maybe look something like the following:

```swift
guard let fileURL = FileManager.default.applicationSupportDirectory?.appendingPathComponent(currentScale.fileName) else {
    return
}

NSWorkspace.shared().open(fileURL)
```